backend/api/migrations/0041_rename_factory_model.py

```diff
@@ -1,16 +1,94 @@
-# Generated by Django 2.2.13 on 2021-10-20 12:37
+# Generated by Django 2.2.13 on 2022-04-23 22:24
```

Comment:
@EagleC0318 I guess you simply copied-pasted the other migration file to make this migration file right? In my opinion, we can build a new table by the "new" ohshown event model. Does this make sense to you?

Reply:
In the other words, we should migrate the model by following the standard django approach. That being said `./manage.py makemigrations` (and then `./manage.py migrate` or rebuild the docker image to initialize the migration files again.

@yanghaochang104 FYI, you may be interested in this tip.